src/components/dashboard/index.js

```diff
@@ -0,0 +1,329 @@
+                }}
+            >
+                <Typography noWrap variant="h6" component="h6">
+                    {content.name}
+                </Typography>
+
+                <Typography
+                    classes={{ root: classes.subtitle }}
+                    gutterBottom
+                    noWrap
+                    color="textSecondary"
+                >
+                    {origination} {`${user} on ${date}`}
+                </Typography>
+
+                <Typography color="textSecondary" variant="body2" component="p">
+                    {description}
+                </Typography>
+            </CardContent>
+
+            <CardActions
+                classes={{
+                    root: classes.actionsRoot,
+                }}
+            >
+                <Button
+                    href={content.link}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    color="primary"
+                >
+                    {getMessage("open")}
+                </Button>
+            </CardActions>
+        </Card>
+    );
+};
+
+const DashboardSection = ({ name, kind, items }) => {
+    const classes = useStyles();
+
+    return (
+        <div className={classes.section}>
+            <Typography
+                noWrap
+                gutterBottom
+                variant="h5"
+                component="h5"
+                color="primary"
+            >
+                {name}
+            </Typography>
+
+            <Divider classes={{ root: classes.dividerRoot }} />
+
+            <div className={classes.sectionItems}>
+                {items.map((item) => (
+                    <DashboardItem kind={kind} content={item} key={item.id} />
+                ))}
+            </div>
+        </div>
+    );
+};
+
+const Dashboard = () => {
+    const classes = useStyles();
+    const [data, setData] = useState({});
+
+    useEffect(() => {
+        const fetcher = async () => {
+            let data = await fetch(
+                `/api/dashboard?limit=${constants.SECTION_ITEM_LIMIT}`,
+                {
+                    method: "GET",
+                    cache: "no-cache",
+                    credentials: "include",
+                    headers: {
+                        "Content-Type": "application/json",
+                    },
+                }
+            )
+                .then((resp) => {
+                    if (resp.status < 200 || resp.status > 299) {
+                        throw new Error(
+                            `${resp.status} ${resp.statusText} ${resp.text()}`
+                        );
+                    }
+                    return resp;
+                })
+                .then((resp) => resp.json())
+                .catch((e) => {
+                    console.log(e);
+                });
+
+            if (!data) {
+                data = {};
+            }
+
+            setData(data);
+        };
+        fetcher();
+    }, []);
```

Comment:
hoping this will be a straightforward `react-query` + axios change. I can look into this after this PR is merged.